create-presigned-download-url/src/main/java/no/unit/nva/download/publication/file/RequestUtil.java

```diff
@@ -0,0 +1,93 @@
+package no.unit.nva.download.publication.file;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import no.unit.nva.download.publication.file.publication.exception.InputException;
+import nva.commons.exceptions.ApiGatewayException;
+import nva.commons.handlers.RequestInfo;
+import org.apache.http.HttpHeaders;
+
+import java.util.Objects;
+import java.util.UUID;
+
+public final class RequestUtil {
+
+    public static final String IDENTIFIER = "identifier";
+    public static final String FILE_IDENTIFIER = "fileIdentifier";
+    public static final String MISSING_AUTHORIZATION_IN_HEADERS = "Missing Authorization in Headers";
+    public static final String IDENTIFIER_IS_NOT_A_VALID_UUID = "Identifier is not a valid UUID: ";
+    public static final String AUTHORIZER_CLAIMS = "/authorizer/claims/";
+    public static final String CUSTOM_FEIDE_ID = "custom:feideId";
+    public static final String MISSING_CLAIM_IN_REQUEST_CONTEXT =
+            "Missing claim in requestContext: ";
+
+    private RequestUtil() {
```

Comment:
Hm, at this point we should refactor these into nva-commons

- getHeader(requestInfo,header)
- getPathParameters(requestinfo, parameter)
- getFromRequestContext(requestInfo, jsonPointer)

Maybe they can even be methods on RequestInfo.

Create  new task on JIRA for it if you don't want to do it as part of this PR

Reply:
I agree, task created. See NP-866.